eodag/resources/providers.yml

```diff
@@ -4471,7 +4471,7 @@
         - dataset
         - $.dataset
       qs: $.qs
-      orderLink: 'https://polytope.lumi.apps.dte.destination-earth.eu/api/v1/requests/destination-earth?{{"verb": "retrieve", "request": {qs#to_geojson} }}'
+      orderLink: 'https://polytope.lumi.apps.dte.destination-earth.eu/api/v1/requests/destination-earth?{{"verb": "retrieve", "request": {qs#to_geojson_polytope} }}'
```

Comment:
use geometry parameter that you convert to _polytope geojson_ instead of converting directly the query string. 

Users should be able to search using `geom` parameter, or pass directly the `feature` in the appropriate polytope format.

Please also add tests